common/src/main/java/com/datastax/oss/simulacron/common/result/AlreadyExistsResult.java

```diff
@@ -31,16 +31,17 @@
   private final String table;
 
   public AlreadyExistsResult(String errorMessage, String keyspace, String table) {
-    this(errorMessage, keyspace, table, 0);
+    this(errorMessage, keyspace, table, 0, false);
   }
 
   @JsonCreator
   public AlreadyExistsResult(
       @JsonProperty("message") String errorMessage,
       @JsonProperty(value = "keyspace", required = true) String keyspace,
       @JsonProperty(value = "table", required = true) String table,
-      @JsonProperty("delayInMs") long delayInMs) {
-    super(ALREADY_EXISTS, errorMessage, delayInMs);
+      @JsonProperty("delayInMs") long delayInMs,
+      @JsonProperty("ignore_on_prime") boolean ignoreOnPrime) {
```

Comment:
I think `ignore_on_prepare` (or `skip_on_prepare`) would be more appropriate to make it more clear that the error is bypassed for the prepare request.